script.js

```diff
@@ -0,0 +1,29 @@
+/////////////////////////////////////////////////
+/* Here I have creat the calculator with the help of 
+1. prompt (work as a calculator screen), 
+2. function (helps to do calculation method), 
+3.console (It's a place where value will be shown.).
+   here when we store the value it's calculate according to there assign calculation method in the function and then after it will get store in array by help of push method.
+*/
```

Comment:
Explanations should be written in the description part of PR. @ManSOorcode  Also you don't have to explain basic things like console!